src/applications/vaos/components/FacilityAddress.jsx

```diff
@@ -61,10 +62,10 @@ export default function FacilityAddress({
           !!phone && (
             <>
               {!!clinicName && <br />}
-              <Heading className="vads-u-font-family--sans vads-u-display--inline vads-u-font-size--base">
-                Main phone:
-              </Heading>{' '}
-              <FacilityPhone contact={phone} />
```

Comment:
Moved to FacilityPhone component.

Reply:
I believe there are other components using FacilityPhone that might need their labels moved.